packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -626,28 +638,31 @@ export const RemoteServerFactory = {
         if (!options.useSLASPrivateClient) {
             return
         }
-        localDevLog(`Proxying ${SLAS_CUSTOM_PROXY_PATH} to ${options.slasTarget}`)
+
+        const slasPrivateClientProxyPath = slasPrivateProxyPath
```

Comment:
we may not need to have a variable if slasPrivateProxyPath is a constant in this case.